src/content/i18n/hi.yml

```
@@ -0,0 +1,124 @@
+a11y.sectionLink: शीर्षक # "Section titled"
+# Site settings
+site.title: Astro दस्तावेज़ीकरण # "Astro Documentation"
+site.og.imageSrc: /default-og-image.png?v=1
+site.og.imageAlt: एक तारों भरे अंतरिक्ष में Astro लोगो, जिसमें दाएँ से एक जामुनी शनि ग्रह की तरह ग्रह तैरता है # "astro logo on a starry expanse of space, with a purple saturn-like planet floating in the right foreground"
+# Left Sidebar
+leftSidebar.sponsoredBy: समर्थित करने वाला # "Sponsored by"
+# Right Sidebar
+rightSidebar.community: समुदाय # "Community"
+rightSidebar.joinDiscord: हमसे Discord में शामिल हों # "Join us on Discord"
+rightSidebar.readBlog: हमारे ब्लॉग पोस्ट पढ़ें # "Read our blog posts"
+rightSidebar.contribute: योगदान करें # "Contribute"
+rightSidebar.translatePage: इस पृष्ठ को अनुवाद करें # "Translate this page"
+# Footer
+footer.privacyPolicy: गोपनीयता नीति # "Privacy Policy"
+# Used in `<Since>`: Added in: v0.24.0 [NEW]
+since.addedIn: 'जोड़ा गया:' # "Added in:"
+since.new: नया # "New"
+since.beta: बीटा # "Beta"
+# Installation Guide
+install.autoTab: स्वचालित CLI # "Automatic CLI"
+install.manualTab: मैन्युअल सेटअप # "Manual Setup"
+# `<DeployGuidesNav>` vocabulary
+deploy.sectionTitle: पृष्ठ संचालन मार्गदर्शिकाएँ # "Deployment Guides"
+deploy.altSectionTitle: अधिक पृष्ठ संचालन मार्गदर्शिकाएँ # "More Deployment Guides"
+deploy.ssrTag: SSR # "SSR"
+deploy.staticTag: Static # "Static"
+# CMS Guides vocabulary
+cms.navTitle: अधिक CMS मार्गदर्शिकाएँ # "More CMS guides"
+# Media Guides vocabulary
+'media.navTitle': 'अधिक DAM मार्गदर्शिकाएँ', # "More DAM guides"
```

Oops, I didn’t look closely. This shouldn’t have the comma and the quotes are also inessential:
```suggestion
media.navTitle: अधिक DAM मार्गदर्शिकाएँ # "More DAM guides"
```